frontend/src/metabase/visualizations/components/List/List.styled.tsx

```diff
@@ -19,6 +19,11 @@ export const Root = styled.div<{ isQueryBuilder?: boolean }>`
     css`
       margin: 2rem 6rem;
     `}
+  td {
```

Comment:
I think this declaration is causing some issues, might be best to remove it for now